apps/mobile/src/components/invitation/KakaoMap/KakaoMap.tsx

```
@@ -0,0 +1,60 @@
+'use client';
+
+import Image from 'next/image';
+import React, { useEffect, useState } from 'react';
+import { CustomOverlayMap, Map } from 'react-kakao-maps-sdk';
```

Biome (1.9.4)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)